### ISSUE TITLE
PlanetShine integration

### ISSUE THREAD

**Issue by DarthPointer** [mockingbirdnest/Principia#2469]
PlanetShine does not work with Principia (tried it on 1.8.1, both up-to-date and previous versions of PS). I have no idea how to contact with PlanetShine contributors.

**Comment by pleroy** [comment]
"Does not work" is not a proper bug report.  Please provide a description of the problem, screenshots, [logs](https://github.com/mockingbirdnest/Principia/wiki/Installing,-reporting-bugs,-and-frequently-asked-questions#reporting-bugs), or other details of the problem.

PlanetShine looks like abandonware to me, so it's hard to get excited about compatibility with that mod.

**Comment by DarthPointer** [comment]
"Does not work" means it does literally nothing. I have done a fresh install with PS. Got really dark nights (no dumb stock ambient light that causes... my eyes hurt if EVE clouds are in) and shadows during the "daylight" looked kinda realistic - as if there was some ambient light source.

I've added Kopernicus, PS kept working. Removed Kopernicus and installed Principia - dumb ambient light got back. Setting it to zero via ingame settings results in absolutelly black shadows even at noon. Looks like Principia disables PS.

Another way to solve my problems is getting another mod, that will remove ambient night light and make shadowed zones lighted with reflected light from other surfaces. I have heard of KS3P, but I can't manage to make it all I need in a proper way. I'm no way a computer graphics spesialist, but I have a powerfull need for Principia and clouds on planets. Clouds with persistent ambient light look not any acceptable.
